### DISCUSSION TITLE
<Help me with the deployment >

### DISCUSSION THREAD

**Discussion by Lalithalavanya** [community/community#65078] [Other Feature Feedback, Questions, & Ideas]
### Deployment 

Show & Tell

### 

Hi this is my first time trying to deploy a project to github pages, whenever I do it just shows a blank page.
But it works fine when I run the html files in the browser without deployment. 
here is the link to the repo:
https://github.com/Lalithalavanya/Travel_website

**Comment by truongtran456** [comment]
What help do you need?

**Comment by arafajs** [comment]
1. Make sure that your repository is named <username>.github.io.
2. In the repository settings, under "Pages", select the "Source" option as "master branch" and the "Branch" option as "gh-pages".
3. In the repository, create a folder called public and put your static website files in that folder.
4. Push your changes to the repository.

Once you have followed these steps, your static website should be deployed to GitHub Pages. If you are still seeing a blank page, there are a few things you can check:

- Make sure that your files are in the public folder.
- Make sure that your files have the correct permissions.
- Try clearing your browser's cache and cookies.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Lalithalavanya ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Comment by seyLu** [comment]
### Github Pages by default looks for `index.html` at the root directory. If it can't find it, it will look for `README.md` instead.

![image](https://github.com/community/community/assets/98249191/7731e18a-a448-4a52-abb4-336c3160f1fa)
![image](https://github.com/community/community/assets/98249191/fc93a41f-559a-4503-8fb1-5afd4fd3a8fe)

ref from [github docs](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site):
> GitHub Pages will look for an `index.html`, `index.md`, or `README.md` file as the entry file for your site.
